package.json

```diff
@@ -43,6 +43,8 @@
   "devDependencies": {
     "@types/jest": "^24.9.1",
     "@types/node": "^12.12.31",
+    "@types/sanctuary": "^0.14.2",
```

Comment:
we are using sanctuary types but not sanctuary itself?

Reply:
good point, I'll remove